test/external/git-process-external-test.ts

```diff
@@ -12,7 +12,11 @@ describe('git-process [with external Git executable]', () => {
   let git: Git | undefined
 
   before(async () => {
-    git = await findGit()
+    try {
+      git = await findGit()
+    } catch (error) {
+      git = undefined
```

Comment:
You could move this up to line 12 and have `let git: Git | undefined = undefined`

Reply:
And the `catch` here can be defined without `error` in TS 2.5 - see https://github.com/desktop/dugite/pull/132 for more info